a1/create_all_output.sh

```diff
@@ -1,8 +1,12 @@
-python a1/main.py -m bayes -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/bayes_1.csv
-python a1/main.py -m bayes -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/bayes_2.csv
-python a1/main.py -m basedt -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/basedt_1.csv
-python a1/main.py -m basedt -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/basedt_2.csv
-python a1/main.py -m perceptron -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/perceptron_1.csv
-python a1/main.py -m perceptron -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/perceptron_2.csv
-python a1/main.py -m basemlp -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/basemlp_1.csv
-python a1/main.py -m basemlp -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/basemlp_2.csv
\ No newline at end of file
+python a1/main.py -m bayes -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/GNB-DS1.txt
+python a1/main.py -m bayes -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/GNB-DS2.txt
+python a1/main.py -m basedt -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Base-DT-DS1.txt
+python a1/main.py -m basedt -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Base-DT-DS2.txt
+python a1/main.py -m bestdt -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Best-DT-DS1.txt
+python a1/main.py -m bestdt -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Best-DT-DS2.txt
+python a1/main.py -m perceptron -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/PER-DS1.txt
+python a1/main.py -m perceptron -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/PER-DS2.txt
+python a1/main.py -m basemlp -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Base-MLP-DS1.txt
+python a1/main.py -m basemlp -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Base-MLP-DS2.txt
+python a1/main.py -m bestmlp -tr dataset/train_1.csv -ts dataset/test_with_label_1.csv -o out/Best-MLP-DS1.txt
+python a1/main.py -m bestmlp -tr dataset/train_2.csv -ts dataset/test_with_label_2.csv -o out/Best-MLP-DS2.txt
```

Comment:
We really do have to have it in a csv file thought.

Reply:
I tried saving it as a csv file, and it worked but the matrix table format was very off

Reply:
Yeah that's why I decided not to put in a csv file. I think whoever wrote the assignment didn't think this through. We could use a separate text file to print the metrics instead. what do you think?